package.json

```diff
@@ -59,15 +59,16 @@
   },
   "devDependencies": {
     "@edx/browserslist-config": "^1.1.1",
-    "@edx/frontend-build": "12.8.10",
     "@edx/reactifex": "^2.1.1",
+    "@openedx/frontend-build": "14.0.3",
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^12.1.5",
     "@testing-library/user-event": "14.4.3",
     "axios": "^1.4.0",
     "axios-mock-adapter": "^1.21.4",
     "glob": "7.2.3",
     "husky": "7.0.4",
-    "jest": "29.5.0"
+    "jest": "29.7.0",
+    "webpack-dev-server": "5.0.4"
```

Comment:
@BilalQamar95 

I'm way late here but do you have any knowledge about why this change was needed? I thought this was previously installed with frontend-build and frontend-build could find and run it's own version. This is causing problems locally because it doesn't match the same version frontend build expects (v4). I can manually update this to v4 but that doesn't fix the issue of why we'd have to manage this manually in here.

I can't remove it because something in this PR causes the original version of webpack installed by frontend-build not to be found anymore.

Reply:
@Zacharis278 Thank you for looping me in on this, this change was needed since `frontend-build` couldn't find and run it's own version. I have since traced and resolved the issue, [fix PR #39](https://github.com/edx/frontend-app-exams-dashboard/pull/39). I have also added you as a reviewer for the PR, kindly review and merge it, thank you.